apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -24,7 +24,7 @@ class LoginTypePicker extends Component {
     const {title, handleLoginChoice, handleCancel} = this.props;
     const googleClassroom = experiments.isEnabled('googleClassroom');
     const microsoftClassroom = experiments.isEnabled('microsoftClassroom');
-    const clever = experiments.isEnabled('clever');
+    const clever = experiments.isEnabled('cleverClassroom');
```

Comment:
I've got that change ready to go, but it builds on top of https://github.com/code-dot-org/code-dot-org/pull/16612 which does all the setup work of hooking this component up to redux (and is a heavy refactor).  If you'd like I can wait a little longer and make these changes part of that PR.